packages/sdk/docker-bake.override.hcl

```diff
@@ -1,3 +1,23 @@
-target "default" {
+target "sdk" {
   tags = ["cartesi/sdk:devel"]
+  labels = {
```

Comment:
this override file is not used by CI, so these labels won't apply there.

Reply:
moved `labels: {}` to `docker-bake.hcl` at da58ebb